src/components/App.js

```diff
@@ -1,31 +1,27 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Calculator from './Calculator';
 import calculate from '../logic/calculate';
 
-export default class App extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
+export default function App() {
```

Comment:
- [ ] Great job you did so far with the refactoring 💯 However, and as we're practicing ES6 features, I highly encourage you to use arrow functions instead of using the `function` keyword 😄 🙏